MongoTrait/Metaable.php

```diff
@@ -36,35 +47,85 @@
     protected $metaFollow = false;
 
     /**
-     * @param string $metaKeywords
+     * @param Collection $metaKeywords
      */
-    public function setMetaKeywords($metaKeywords)
-    {
+    public function setMetaKeywords(Collection $metaKeywords) {
         $this->metaKeywords = $metaKeywords;
```

Comment:
@djamnazi : you need to refacto this part to check the pre-conditions. You can't store something that's not a metakeyword in the collection